cli_client/python/timesketch_cli_client/commands/timelines.py

```diff
@@ -35,14 +35,14 @@ def list_timelines(ctx):
 
 
 @timelines_group.command("describe")
-@click.argument("timeline_id", type=int, required=False)
+@click.argument("timeline-id", type=int, required=False)
```

Comment:
@berggren  as discussed I fixed it here to have propper argument names but be aware of: https://click.palletsprojects.com/en/7.x/options/#name-your-options

Reply:
yes, they are converted to underscores for the python name.